.circleci/config.yml

```diff
@@ -52,23 +81,40 @@ pip-run:  &pip-install
   # Upgrade pip and setuptools and wheel to get as clean an install as possible
   name: Upgrade pip, setuptools, wheel
   command: |
-    pip install --upgrade --user pip
-    pip install --upgrade --user wheel
-    pip install --upgrade --user setuptools
+    export PATH="$HOME/miniconda/bin:$PATH"
+    source activate testenv
+    pip install --upgrade pip
+    pip install --upgrade wheel
+    pip install --upgrade setuptools
 
 deps-run: &deps-install
   name: Install Python dependencies
   command: |
-    pip install --user python-dateutil numpy${NUMPY_VERSION} pyparsing!=2.1.6 cycler codecov coverage sphinx pillow
-    pip install --user -r doc-requirements.txt
+    export PATH="$HOME/miniconda/bin:$PATH"
+    source activate testenv
+    pip install python-dateutil numpy${NUMPY_VERSION} pyparsing!=2.1.6 cycler codecov coverage
+    pip install -r doc-requirements.txt
 
 mpl-run: &mpl-install
   name: Install Matplotlib
-  command: pip install --user -ve .
+  command: |
+    export PATH="$HOME/miniconda/bin:$PATH"
+    source activate testenv
+    pip install -ve .
```

Comment:
do the docs build with a non-editable installation?

Reply:
They should. I can switch to `pip install -v .` if you prefer.